package.json

```diff
@@ -37,10 +37,10 @@
     "node-dir": "^0.1.17"
   },
   "dependencies": {
-    "babel-preset-es2016": "^6.24.1",
     "jquery": "^3.5.0",
```

Comment:
Are we still keeping jquery as a dependency?

Reply:
jQuery is needed for the CMS side of things, but not for the front end.